src/components/framework/footer.js

```diff
@@ -1,251 +1,135 @@
-            Japan National Institute of Infectious Diseases</a>.
-          </div>
-          <div style={styles.acknowledgments}>
-            Special thanks to Jackie Katz, Dave Wentworth, Becky Garten, Vivien Dugan, Xiyan Xu,
-            Elizabeth Neuhaus, Sujatha Seenu, John McCauley, Rod Daniels, Vicki Gregory, Kanta
-            Subbarao, Ian Barr, Aeron Hurt, Takato Odagiri, Shinji Watanabe, Tomoko Kuwahara,
-            Michael Lässig, Marta Łuksza, Richard Reeve, Colin Russell, Sebastian Maurer-Stroh and
-            Peter Bogner for feedback and advice. This analysis represents an updated frontend to
-            the Nextflu informatic pipeline, originally available at nextflu.org.
-          </div>
-          <div style={{display: "flex", justifyContent: "space-between"}}>
-            <div style={{flex: 5}}/>
-            <a key={5} href="https://gisaid.org" target="_blank" rel="noreferrer noopener" style={{display: "flex", flexDirection: "column", justifyContent: "center"}}>
-              <img alt="gisaid-logo" width="120" src={require("../../images/gisaid-logo.png")}/>
-            </a>
-            <div style={{flex: 1}}/>
-            <a key={6} href="http://www.who.int/influenza/gisrs_laboratory/en/" target="_blank" rel="noreferrer noopener">
-              <img alt="gisrs-logo" width="120" src={require("../../images/gisrs-logo.jpg")}/>
-            </a>
-            <div style={{flex: 5}}/>
-          </div>
-        </div>
-      );
-    }
+  .finePrint {
+    font-size: 14px;
   }
 
-  return (
-    <div>
-      {genericPreamble}
-    </div>
-  );
+  .acknowledgments {
+    margin-top: 10px;
+  }
+
+  .filterList {
+    margin-top: 10px;
+    line-height: 1.0;
+  }
+
+  .imageContainer {
+    display: flex;
+    justify-content: space-around;
+    align-items: center;
+  }
+`;
+
+export const getAcknowledgments = (metadata, dispatch) => {
+  /**
+   * If the metadata contains a description key, then it will take precendence the hard-coded
+   * acknowledgements. Expects the text in the description to be in Mardown format.
+   * Jover. December 2019.
+  */
+  if (metadata.description) {
+    dompurify.addHook("afterSanitizeAttributes", (node) => {
+      // Set external links to open in a new tab
+      if ('href' in node && location.hostname !== node.hostname) {
+        node.setAttribute('target', '_blank');
+        node.setAttribute('rel', 'noreferrer nofollow');
+      }
+      // Find nodes that contain images and add imageContainer class to update styling
+      if (([...node.childNodes].filter((child) => child.localName === 'img')).length > 0) {
+        node.className += ' imageContainer';
+      }
+    });
+
+    const sanitizer = dompurify.sanitize;
+    const sanitizerConfig = {
+      ALLOWED_TAGS: ['div', 'h1', 'h2', 'h3', 'h4', 'h5', 'h6', 'p', 'em', 'strong', 'del', 'ol', 'ul', 'li', 'a', 'img', '#text'],
+      ALLOWED_ATTR: ['style', 'href', 'src', 'width', 'height', 'alt'],
```

Comment:
Note that allowing `style` will allow clickjacking. Is it absolutely necessary, especially considering images are now put into an `.imageContainer`?

Reply:
Ah, right. I will remove it. I guess if people really want to customize their footer image styling, then they can change the styling within Auspice rather than the Markdown file.